doc/getting-started/install.md

```diff
@@ -39,12 +39,13 @@ sudo yum install systemd-devel ncurses-devel ncurses-compat-libs -y
 
 For Debian/Ubuntu, use the following instead:
 
-```bash
-sudo apt-get update -y
-sudo apt-get install automake build-essential pkg-config libffi-dev libgmp-dev libssl-dev libtinfo-dev libsystemd-dev zlib1g-dev make g++ tmux git jq wget libncursesw5 libtool autoconf -y
+
```

Comment:
@soyfrien Why have you removed ```bash?

Reply:
Oops, I am going to close this, it's many commits behind now and it's not a change that makes a difference to any outcomes. I'll add it in if I ever make a bigger pull request.